src/test/util-test.ts

```diff
@@ -35,4 +35,12 @@ describe('util', () => {
 			assert.equal(matches, true);
 		});
 	});
+	describe('isGlobalTSFile()', () => {
+		it('should match declaration files in node_modules', () => {
+			assert.equal(isGlobalTSFile('/node_modules/@types/not_node/not_node.d.ts'), true);
```

Comment:
Ah, right these should match another `import` elsewhere. 
I will remove this case and better document cases where the pre-loading fails (if it still does).

The tsdlib.d.ts is not referenced in source (only ProjectConfiguration can know from checking options.importHelpers) - is it okay if it stays in the isGlobalTSFile check?